src/test/regress/expected/with_transactions.out

```diff
@@ -85,17 +85,18 @@ UPDATE raw_table SET created_at = '2001-02-10 20:00:00' WHERE tenant_id IN (SELE
 DEBUG:  common table expressions are not supported in distributed modifications
```

Comment:
Why did this file change? I assume this is with non-adaptive executor and its output should stay the same?